connectors/src/connectors/slack/lib/workspace_limits.ts

```diff
@@ -1,17 +1,15 @@
 import type { WorkspaceDomain } from "@dust-tt/types";
 import { cacheWithRedis, DustAPI } from "@dust-tt/types";
-import type { UsersInfoResponse, WebClient } from "@slack/web-api";
-import type {
-  Profile,
-  User,
-} from "@slack/web-api/dist/response/UsersInfoResponse";
+import type { WebClient } from "@slack/web-api";
+import type {} from "@slack/web-api/dist/response/UsersInfoResponse";
 
+import type { SlackUserInfo } from "@connectors/connectors/slack/lib/slack_client";
 import { getSlackConversationInfo } from "@connectors/connectors/slack/lib/slack_client";
 import { dataSourceConfigFromConnector } from "@connectors/lib/api/data_source_config";
 import logger from "@connectors/logger/logger";
 import type { ConnectorResource } from "@connectors/resources/connector_resource";
 
-const WHITELISTED_BOT_NAME = ["Beaver", "feedback-hackaton", "Dust"];
+const WHITELISTED_BOT_NAME = ["Beaver", "feedback-hackaton", "Dust", "Retool"];
```

Comment:
Made a second review. Once the other question is resolved. Let's add a comment here to warn about the implication:

```
Whitelisting a bot will accept any message from this bot. This means that we rely on the bot security measures to not ping Dust from actions done by non verified members of the workspace. Make sure to be explicit about this with users as you whitelist a new bot.
```